ultravox/model/ultravox_model.py

```diff
@@ -529,6 +544,39 @@ class ModifiedWhisperEncoder(
     base_model_prefix = "model.encoder"
     _no_split_modules = ["WhisperEncoderLayer"]
 
+    def init_latency_mask(self, audio_latency_block_size: int, dtype: torch.float):
```

Comment:
Is this what you meant?
```suggestion
    def init_latency_mask(self, audio_latency_block_size: int, dtype: torch.dtype):
```